workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/SpatialOSComponent.cs

```diff
@@ -36,5 +36,24 @@ public bool TryGetGameObjectForSpatialOSEntityId(EntityId entityId, out GameObje
             linkedGameObject = entityManager.GetComponentObject<GameObjectReference>(entity).GameObject;
             return true;
         }
+
+        public bool TryGetSpatialOSEntityIdForGameObject(GameObject linkedGameObject, out EntityId entityId)
```

Comment:
nitpick - any gameobject is a valid argument here and a user doesn't always know before calling that it is linked. So the name is slightly misleading.